.github/workflows/regression-tests.yml

```diff
@@ -31,7 +31,7 @@ jobs:
 
       - name: Configure & Build
         run: |
-          apt update && apt install -y pip
+          apt update && apt install -y pip  && apt install -y jq
```

Comment:
small nit -
```suggestion
          apt update && apt install -y pip jq
```

Reply:
changing it to apt update && apt install -y pip jq